# answers for part 1 
1. `` values added: 20``
2. `` final result: 20``
3. ``values added: 20``
4. result is not defined since it is out of scope. Because let only has block scope
5. It throws an error because we are trying to reassign a value to a const variable.
6. The same error persists, and nothing is printed because we are reassigning a const variable
